shell/pom.xml

```diff
@@ -35,16 +35,17 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
```

Comment:
guava is dependency managed. See `dependencyManagement` section in parent `pom.xml`.  For those dependencies, we don't need to specify their version. To get cleaner pom structure, we could either put all external dependencies into `dependencyManagement`, or we could remove `dependencyManagement` and always use defined version constant. Right now we are mixing these two approaches. I incline for the former one.